src/components/Driver/DriverProfile.js

```diff
@@ -0,0 +1,79 @@
+import React, {Component} from "react"
+import axios from 'axios'
+import ProfileInfoTable from '../../utils/ProfileInfoTable'
+import Loading from '../../utils/Loading'
+
+class DriverProfile extends Component {
```

Comment:
I suggest keeping functional approach of writing components

Reply:
Its changed.